lib/cli/config.js

```diff
@@ -18,12 +18,15 @@ const configDefault = {
 
 const configValidate = config => {
   const result = Joi.validate(config, JoiConfigSchema);
+
   if (result.error) return result;
+
   if (!Array.isArray(config.order)) {
     return {
-      error: new ValidationError('empty order property'),
+      error: new ValidationError('Empty order property.'),
```

Comment:
Not sure what the common pattern is for error messages, in regards to ending with a `.` or not.